### DISCUSSION TITLE
Fauxmo Alexa switch

### DISCUSSION THREAD

**Discussion by David-Gelles** [vintlabs/fauxmoESP#210] [Q&A]
Hey guys,
I`m controlling all my lights with fauxmo on esp8266... working realy great.

So i wanted to expand my system and build a remote control including esp8266 and some switches.
Those switches are going to be set as an input to change the state of every device added to alexa
Now there is my question:
 Is that possible?

**Comment by bm-a** [comment]
yes you can do it i am not so good at coding but i was able to do it using tasmota
![NodeMCU-Blynk-Home-Automation-3](https://user-images.githubusercontent.com/78638417/145679373-ba460ed9-917e-4da3-8243-218f6af5159d.jpg)
![image](https://user-images.githubusercontent.com/78638417/145679425-8af4324f-0c3d-459f-bf85-602db2172fcc.png)
 this is my tasmota config.
you can find out the code for it.
